api/src/main/java/org/apache/cloudstack/api/command/admin/network/UpdateNetworkOfferingCmd.java

```diff
@@ -55,7 +55,7 @@ public class UpdateNetworkOfferingCmd extends BaseCmd {
     private String displayText;
 
     @Parameter(name = ApiConstants.AVAILABILITY, type = CommandType.STRING, description = "the availability of network offering."
-            + " Default value is Required for Guest Virtual network offering; Optional for Guest Direct network offering")
+            + " The default value is Required for Guest Virtual network offering, Optional for Guest Direct network offering. Only one network offering can have the value Required ")
```

Comment:
```suggestion
            + " The value is Required makes this network offering default for Guest Virtual Networks. Only one network offering can have the value Required ")
```
I do not know the significance of the term `Guest Direct network offering` in the description so I am proposing this change @weizhouapache @kiranchavala . Please enhance if needed.

Reply:
ping @kiranchavala , can you approve of my suggestion?

Reply:
@DaanHoogland Yes suggestion is fine